torchx/schedulers/test/kubernetes_mcad_scheduler_test.py

```diff
@@ -337,6 +359,30 @@ def test_role_to_pod(self) -> None:
                             medium="Memory",
                         ),
                     ),
+                    V1Volume(
+                        name="odh-trusted-ca-cert",
+                        config_map=V1ConfigMapVolumeSource(
+                            name="odh-trusted-ca-bundle",
+                            items=[
+                                V1KeyToPath(
+                                    key="ca-bundle.crt", path="odh-custom-ca-bundle.crt"
```

Comment:
While I was testing, I noticed the path needs to exactly match the subpath afaict

Reply:
Updated the volumes to match now